### PR TITLE
Switch to project default at the beginning of each test.

### PR DESCRIPTION
**Pull request by phracek**
Switch to project default at the beginning of each test.

### CODE REVIEW

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
Rebased on the top of master

[test]

**Comment by zmiklank** [comment]
I see that this change is for the openshift tests, so I will run also the openshift testsuite.
[test-openshift]

File: test-lib.sh [hunk 1]
```diff
@@ -1310,6 +1310,11 @@ ct_run_tests_from_testset() {
         TESTSUITE_RESULT=1
       fi
     fi
+    # As soon as test is finished
+    # switch the project from sclorg-test-<NUMBER> to default.
+    if [ "${CT_OCP4_TEST:-false}" == "true" ]; then
```
Comment (zmiklank) [inline test-lib.sh:1315]: Handling OC stuff does not really seem to fit to this function.  It is not related to executing tests.
However, I understand, that adding this line to all of the containers is not really great solution too.

What about adding these lines to the openshift cleanup function `ct_os_cleanup`?

**Review by zmiklank — CHANGES_REQUESTED** [review]

File: test-lib.sh (hunk 1 above)
Comment (phracek) [reply test-lib.sh:1315]: This function has to be called after each test and not after the whole test suite.

Comment (zmiklank) [reply test-lib.sh:1315]: Oh, I see now, thanks for explanation. I am OK with this solution then.

**Review by zmiklank — APPROVED** [review]

**Pull request merged**